damus/Views/Profile/ProfileView.swift

```diff
@@ -103,6 +103,33 @@ struct ProfileView: View {
         return Double(-yOffset > navbarHeight ? progress : 0)
     }
     
+    func getProfileInfo() -> (String, String) {
+        let profile_txn = self.damus_state.profiles.lookup(id: profile.pubkey)
+        let ndbprofile = profile_txn?.unsafeUnownedValue
+        let displayName = Profile.displayName(profile: ndbprofile, pubkey: profile.pubkey).displayName.truncate(maxLength: 25)
+        let userName = Profile.displayName(profile: ndbprofile, pubkey: profile.pubkey).username.truncate(maxLength: 25)
+        return (displayName, "@\(userName)")
+    }
+    
+    func showFollowBtnInBlurrBanner() -> Bool {
+        damus_state.contacts.follow_state(profile.pubkey) == .unfollows && bannerBlurViewOpacity() > 1.0
+    }
+    
+    func bannerTextView(minY: CGFloat) -> some View {
+        VStack(alignment: .leading, spacing: -4.5) {
+            Text("\(getProfileInfo().0)")
+                .font(.headline)
+                .foregroundColor(.white)
+            Text("\(getProfileInfo().1)")
+                .font(.subheadline)
+                .foregroundColor(.white.opacity(0.8)) // slightly dimmer for distinction
```

Comment:
This will be invisible in profiles where the profile banner is in a bright color or white. Example:

<img width="249" alt="Screenshot 2024-12-03 at 5 40 55 PM" src="https://github.com/user-attachments/assets/60a8c770-46fe-401f-a134-c1ca0e10cab4">

We should make this color dynamically computed based on the background, or perhaps we should add a darkening effect on the banner in `VisualEffectView(effect: UIBlurEffect(style: .systemUltraThinMaterial)).opacity(bannerBlurViewOpacity())`

Reply:
Thanks for the feedback. I used ToolbarItem for displaying the Profile info text. Also, I have used darkening effect in `VisualEffectView` in order to resolve the color blending issue.

Reply:
I've updated the PR description as per the recent changes